content/docs/authorization/v2.x/configuration/powermax/_index.md

```diff
@@ -116,12 +116,12 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
         # enabled: Always set to true
         enabled: true
         forceRemoveModule: true
-        configVersion: v2.12.0
+        configVersion: v2.11.0
         components:
         - name: csipowermax-reverseproxy
         # image: Define the container images used for the reverse proxy
         # Default value: None
-          image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.12.0
+          image: quay.io/dell/container-storage-modules/csipowermax-reverseproxy:v2.11.0
```

Comment:
Are we downgrading (or) wrongly updated before? Is this the latest version of reverse proxy image?

Reply:
Last version for Reverse Proxy was v2.10.0, so it should be v2.11.0 for csipowermax-reverseproxy.